Sports/live_nba.1m.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env PYTHONIOENCODING=UTF-8 /usr/local/bin/python3
+
+# <bitbar.title>Live NBA Game Stat</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Chen Luxin</bitbar.author>
+# <bitbar.author.github>chenlujjj</bitbar.author.github>
+# <bitbar.desc>Show NBA game scores in bar.</bitbar.desc>
+# <bitbar.dependencies>python3</bitbar.dependencies>
+
```

Comment:
Any chance you could add an image here?

Reply:
sure, I just added an image to show the plugin effect.